spec/models/patient_phone_number_spec.rb

```diff
@@ -103,4 +103,21 @@
         .from(0).to(1)
     end
   end
+
+  describe "#number_with_country_code" do
+    it "adds the country code when one is not present" do
```

Comment:
Can you add some more test cases here ? What happens when its some sort of invalid phone number, like not enough digits or too many?

Reply:
I added more tests. In the process, I found some behavior in the gem I don't totally understand regarding how it strips the raw national number. I'm guessing it may have something to do with what constitutes a "valid" number. If the abbreviation is set to US and you have a 9 digit number that starts with 1, it returns the number you gave it. With numbers of other lengths, it strips the leading 1. So "111-111-111" and "111-111-1111" both return nine ones. I don't think it's worth digging into further. The truth is the current process for processing phone numbers is imperfect and we're not going to fix it today.